frontend/src/app/feature/dashboard/dashboard.component.ts

```diff
@@ -1,29 +1,38 @@
 import {Component, HostListener, OnDestroy, OnInit} from '@angular/core';
 import {Router} from '@angular/router';
-import {DashboardService} from '../../core/service/dashboard.service';
+import {RoomService} from '../../core/service/room.service';
 import {WithWebSocket} from '../../util/web-socket/with-web-socket';
 import {WsMessage} from '../../model/ws-message';
 import {WsDashboardMessageType} from '../../util/web-socket/ws-message-type';
 import {UserService} from '../../core/service/user.service';
-import {UuidUtil} from '../../util/uuid.util';
-import {WsConstants} from '../../util/web-socket/ws-constants';
+import {Generator} from '../../util/generator';
+import {Room} from '../../model/room';
+import {BehaviorSubject} from 'rxjs';
+import {tap} from 'rxjs/operators';
+import {ApiConst} from '../../util/api.const';
 
 @Component({
   selector: 'dashboard',
   templateUrl: './dashboard.component.html',
   styleUrls: ['./dashboard.component.scss']
 })
 export class DashboardComponent extends WithWebSocket implements OnInit, OnDestroy {
+
+  private rooms: Array<Room> = [];
+  private roomsSbj = new BehaviorSubject<Array<Room>>(this.rooms);
```

Comment:
Why not just rooms?  angular will notice change in collection anyway

Reply:
I will think about it. There are some differences but I need to check them. Let's skip it for now. Will get back to this later